native/dlassq.go

```diff
@@ -0,0 +1,25 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlassq updates a sum of squares in scaled form.
+func (impl Implementation) Dlassq(n int, x []float64, incx int, scale *float64, sumsq *float64) {
```

Comment:
Remove redundant types. Add explanation of parameters. Can we instead return scale and sumsq?